src/apps/bridge/sensorController.cpp

```diff
@@ -18,6 +20,8 @@
 #define DEFAULT_CURRENT_READING_PERIOD_MS 60 * 1000       // default is 1 minute: 60,000 ms
 #define DEFAULT_SOFT_READING_PERIOD_MS 500                // default is 500 ms (2 HZ)
 #define DEFAULT_SEAPOINT_TURBIDITY_READING_PERIOD_MS 1000 // default is 1 second: 1000 ms (1 HZ)
+#define DEFAULT_PME_DISSOLVED_OXYGEN_READING_PERIOD_MS 10 * 60 * 1000 // 10 minutes
+#define DEFAULT_PME_WIPER_READING_PERIOD_MS 10 * 60 * 1000            // 10 minutes
```

Comment:
Default wipe should be 4 hours per the `wipeIntervalMin` config in the spec. PME hasn't implemented the config yet, but we did decide that the default would be 4 hours.